### ISSUE TITLE
Fetch default content type

### ISSUE THREAD

**Issue by jcf120** [shortercode/Doom.js#23]
Alex is pushing for default `'Content-type': 'text/plain'` for the sake of removing options requests. Networking's a bit voodoo to me, reckon it's safe? Sounds like it might be the norm:
https://geonet.esri.com/thread/8170

**Comment by shortercode** [comment]
Yeah I've wanted to change it to plain text for awhile but I know the
paddock app depended on that behaviour.

**Comment by jcf120** [comment]
Okey doke, I'll just add
```javascript
Doom.fetch.DEFAULTS.headers['Content-type'] = 'application/json';
```
to the top of paddock, and change Doom's internal one to `text/plain`.

**Comment by jcf120** [comment]
Done: 94fbe6e27ffe9d155875c044b072d99944088af2
